### PR TITLE
Unit test for Recipe class

### PR DESCRIPTION
**Pull request by Sam-Burns**
Adds a unit test for `Symfony\Flex\Recipe`

### CODE REVIEW

File: tests/RecipeTest.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Tests;
+
+use Composer\Package\Package;
+use PHPUnit_Framework_MockObject_MockObject as Mock;
```
Comment (stof) [inline tests/RecipeTest.php]: we never add use statements for classes in the global namespace

File: tests/RecipeTest.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+     * @var Recipe
+     */
+    private $recipe;
+
+    /**
+     * @var Package|Mock
+     */
+    private $package;
+
+    public function setUp()
```
Comment (stof) [inline tests/RecipeTest.php]: should be protected

File: tests/RecipeTest.php (hunk 1 above)
Comment (TomasVotruba) [inline tests/RecipeTest.php]: @stof Just curious, is there a fixer for that to automate this in reviews?

Comment (stof) [inline tests/RecipeTest.php]: I don't think PHP-CS-Fixer supports it currently.

File: tests/RecipeTest.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+class RecipeTest extends TestCase
+{
+    /**
+     * @var Recipe
+     */
+    private $recipe;
+
+    /**
+     * @var Package|PHPUnit_Framework_MockObject_MockObject
```
Comment (maxhelias) [inline tests/RecipeTest.php]: I think, it should be : `@var Package|PHPUnit\Framework\MockObject\MockObject`

File: tests/RecipeTest.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+     */
+    private $package;
+
+    protected function setUp()
+    {
+        $this->package = $this->getMockBuilder(Package::class)->disableOriginalConstructor()->getMock();
+        $this->recipe = new Recipe(
+            $this->package,
+            'name',
+            [
```
Comment (maxhelias) [inline tests/RecipeTest.php]: The `job` parameters are missing : https://github.com/symfony/flex/blob/master/src/Recipe.php#L26

File: tests/RecipeTest.php (hunk 3 above)
Comment (localheinz) [reply tests/RecipeTest.php]: Actually, it should be `@var Package|\PHPUnit\Framework\MockObject\MockObject` (note the missing leading `\`).

**Pull request closed without merging**